### DISCUSSION TITLE
Please credit the original torchaudio tutorials.

### DISCUSSION THREAD

**Discussion by mthrok** [deepgram/community#85] [General help] [answered]
Hi 

We found that the following blog is based on our tutorials.

https://blog.deepgram.com/pytorch-intro-with-torchaudio/

https://pytorch.org/audio/0.10.0/tutorials/audio_io_tutorial.html
https://pytorch.org/audio/0.10.0/tutorials/audio_data_augmentation_tutorial.html
https://pytorch.org/audio/0.10.0/tutorials/audio_feature_extractions_tutorial.html

We appreciate if you give a credit to the original work and reference to the source.

Thanks,

**Comment by lukeocodes** [comment, marked as answer]
Hey moto, thanks for contacting us.

I believe you’ve contacted us before about this, and it was my understanding that we responded. Apologies if you didn’t receive it. We believe there is extra context to the content, and that simply having code samples is not sufficient reason to call this plagiarism. We offer links to PyTorch in the articles to provide attribution and mention PyTorch and Torch Audio throughout the article.

All this being said, we acknowledge your perception on this and greatly appreciate all the work you have done for PyTorch.

We take plagiarism very seriously with our guest authors, and check all submitted content against grammarly for examples of plagiarism. In this example, the only duplicate content we found between this post and your documentation are the code snippets, where our writer has used your guides in his development, and then later in his production of this content.

To help resolve this concern, we’ll be adding further-reading links to the content you’ve mentioned here.
